devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java

```diff
@@ -135,6 +137,50 @@ public PolicyContract() {
     @CallFlags(All)
     public native void setStoragePrice(int price);
 
+    /**
+     * @return the block generation time in milliseconds
+     */
+    @CallFlags(ReadOnly)
+    public native int getMillisecondsPerBlock();
+
+    /**
+     * Sets the block generation time in milliseconds.
+     *
+     * @param milliseconds The block generation time in milliseconds.
+     */
+    @CallFlags(States | AllowNotify)
```

Comment:
feat: as discussed in the scope of #1104, let's use more loose call flags, i.e., `All` for setters. Devs can still extend this class, override a function and apply more strict call flags if required for some reason. Please update appropriately for all 3 new setter functions here.